keywords/LiteServiOS.py

```diff
@@ -1,39 +1,80 @@
 import json
 import subprocess
+import os
+import re
+import time
+from zipfile import ZipFile
+from shutil import copyfile
+import requests
 
 from keywords.LiteServBase import LiteServBase
 from keywords.constants import BINARY_DIR
+from keywords.constants import LATEST_BUILDS
 from keywords.exceptions import LiteServError
 from keywords.utils import version_and_build
 from keywords.utils import log_info
 from keywords.utils import log_r
+from keywords.constants import REGISTERED_CLIENT_DBS
+from keywords.constants import CLIENT_REQUEST_TIMEOUT
 
 
 class LiteServiOS(LiteServBase):
 
     def __init__(self, version_build, host, port, storage_engine):
-
-        # Initialize baseclass properies
         super(LiteServiOS, self).__init__(version_build, host, port, storage_engine)
```

Comment:
add a check for `forestdb` and `forestdb+encryption`. If the user tries to use these with iOS, raise an exception.